### ISSUE TITLE
fpic : can't go past make 64 bit fedora os

### ISSUE THREAD

**Issue by 3rdshiftcoder** [slacka/WoeUSB#151]
Hi-

I am using fedora 25 xfce 64 bit. I got the dreaded -fpic compile error. I think it has to do with 32 and 64 bit software not coexisting nicely.  ver 2.2 of software with new configure creation process.
My errors are: 
/usr/bin/ld: MainPanel.o: relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
Just wondering, is there an easy flag or fix I can set when I configure to make this message go away when I run make? I installed both fedora dependencies and wxGTK-devel.
I could try using 32 bit but I am on a 64 bit os. 
I need a rebootable usb from linux for win 10 because I am trying to reset my win10 password. I got locked out of my local admin account. I didn't have an active default administrator account and didn't have the microsft account option active at log-in. Will try to use pcunlocker after making bootable usb 10 flash drive. ISO2disc didn't populate with usb-drive in combobox.
thx. so much - J. McNamara

**Comment by 3rdshiftcoder** [comment]
looks like a versioning and patch issue between wxwidgets 2.8 and 3.0. I will try to revert. I just used whatever wxGTK that is in fedora repo.
Will post confirmation when I try to correct in a minute. thx. J.

**Comment by Lin-Buo-Ren** [comment]
Hello, as a workaround  you may directly run the `src/woeusb` script in the source tree to build a Windows USB device

**Comment by 3rdshiftcoder** [comment]
Hi-
I figured out by reading the notes in the issues. My problem was the more current version requires wxGTK3-devel. I thought I had that but in fact had wxGTK-devel-2.8.x I found out what is on my system by runninig dnf history userinstalled. Then I removed 2.8 and put on 3.0 version. I launched the app from my applications menu in xfce. I will close. That fixed my mistake in versioning. - thx.- J. McNamara
